unstructured/ingest/cli/cmds/airtable.py

```diff
@@ -1,76 +1,107 @@
 import logging
+import typing as t
```

Comment:
are we importing just for the sake of importing Optional? any reason to not just import Optional from typing?

Reply:
I found this pattern widely used in other packages and it helps in files with a lot of typing imports. I wanted to start migrating to this.